test/Faker/Provider/en_GB/CompanyTest.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace Faker\Provider\en_GB;
+
+use Faker\Test\TestCase;
```

Comment:
The namespace is wrong.

```suggestion
namespace Faker\Test\Provider\en_GB;

use Faker\Provider\en_GB\Company;
use Faker\Test\TestCase;
```